README.md

```diff
@@ -1 +1,29 @@
-# team-culture
\ No newline at end of file
+# Code of Conduct
+
+## Diversity
+We accept others, no matter the gender, nationality, culture, religion.
+
+## Respect
+We all have different backgrounds, beliefs and points of view.
```

Comment:
This feels like the duplicate of the first one.

Reply:
Maybe, but we should remark both of them. Maybe we should add something else here.